src/js/providers/html5.js

```diff
@@ -130,14 +130,15 @@ define([
 
         // Find video tag, or create it if it doesn't exist.  View may not be built yet.
         var element = document.getElementById(_playerId);
-        var _videotag = (element) ? element.querySelector('video') : undefined;
+        var tagType = _playerConfig.useAudioTag ? 'audio' : 'video';
+        var _videotag = (element) ? element.querySelector(tagType) : undefined;
```

Comment:
If we are going to query for an existing tag, we should query for video or audio. So we only need to look at the config when creating a media element, and at that point create an video or audio element.